auth_test.go

```diff
@@ -85,56 +88,131 @@ func (ts *AuthTests) Test_login_only_once() {
 	ts.HandleRequestError(req, "permission denied")
 }
 
+// A task that is not yet running can't log in.
+func (ts *AuthTests) Test_login_staging_task() {
+	ts.SetupBackendWithMesos()
+	ts.AddTask(mkTask("staging", "staging-task.abc-123", mesos.TASK_STAGING))
+	ts.SetTaskPolicies("staging-task", "insurance")
+	req := ts.mkReq("login", jsonobj{"task-id": "unregistered-task.abc-123"})
+	ts.HandleRequestError(req, "permission denied")
```

Comment:
I'm wondering if there's any precedent for more informational error responses? ...Issue for another PR, tho.

Reply:
I'd rather not reveal too much information in authentication failure responses. I don't really know what a good balance is, though.